x/protorev/client/cli/query.go

```diff
@@ -148,6 +149,14 @@ func NewQueryPoolWeightsCmd() (*osmocli.QueryDescriptor, *types.QueryGetProtoRev
 	}, &types.QueryGetProtoRevPoolWeightsRequest{}
 }
 
+// NewQueryPoolCmd returns the command to query the pool id for a given denom pair
+func NewQueryPoolCmd() (*osmocli.QueryDescriptor, *types.QueryGetProtoRevPoolRequest) {
+	return &osmocli.QueryDescriptor{
+		Use:   "pool [base_denom] [other_denom]",
+		Short: "Query the pool id for a given denom pair stored in ProtoRev",
```

Comment:
Update this based on the comment above.

Reply:
yeah, just want highest liquidity pools. Specified here: https://github.com/osmosis-labs/osmosis/pull/4829/commits/821b247669016401caec222de0743718f194e4e8